client/src/modules/InterviewQuestions/pages/InterviewQuestionsPage/index.tsx

```diff
@@ -0,0 +1,120 @@
+import { Col, Layout, message, Row, Spin, Tabs } from 'antd';
+import { AdminSider, Header, Session } from 'components';
+import { AddCategoryButton } from 'modules/InterviewQuestions/components/Buttons/AddCategoryButton';
+import { AddQuestionButton } from 'modules/InterviewQuestions/components/Buttons/AddQuestionButton';
+import { CategoryModalForm } from 'modules/InterviewQuestions/components/Modals/CategoryModalForm';
+import { QuestionsModalForm } from 'modules/InterviewQuestions/components/Modals/QuestionModalForm';
+import { CategoriesTable } from 'modules/InterviewQuestions/components/Tables/CategoriesTable';
+import { QuestionsTable } from 'modules/InterviewQuestions/components/Tables/QuestionsTable';
+import { useState } from 'react';
+import { InterviewQuestionCategoryService, InterviewQuestionService } from 'services/interviewQuestion';
+import { InterviewQuestion, InterviewQuestionCategory } from 'services/models';
+
+const { Content } = Layout;
+const { TabPane } = Tabs;
+type Props = { session: Session; questions: InterviewQuestion[]; categories: InterviewQuestionCategory[] };
+
+export function InterviewQuestionsPage(props: Props) {
+  const [questions, setQuestions] = useState(props.questions);
+  const [categories, setCategories] = useState(props.categories);
+  const [modalQuestionData, setModalQuestionData] = useState(null as InterviewQuestion | null);
```

Comment:
Better use `useState<InterviewQuestion | null>(null)` instead. Because `null` is not `InterviewQuestion`.